src/navigation-plan.js

```diff
@@ -1,8 +1,10 @@
 import {Redirect} from './navigation-commands';
 
-export var NO_CHANGE = 'no-change';
-export var INVOKE_LIFECYCLE = 'invoke-lifecycle';
-export var REPLACE = 'replace';
+export const activationStrategy = {
```

Comment:
I think we should keep it singluar. I would rather update the logger to match that.